apps/nextjs-website/src/components/atoms/InnerHtmlLazyLoadedPart/InnerHtmlLazyLoadedPart.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import { Box, useTheme } from '@mui/material';
+
+export type InnerHtmlLazyLoadedPartProps = { html: string };
+
+const InnerHtmlLazyLoadedPart = ({ html }: InnerHtmlLazyLoadedPartProps) => {
+  const { spacing, palette } = useTheme();
+  const [loaded, setLoaded] = useState(false);
+  useEffect(() => setLoaded(true), []);
+
+  return (
+    <>
+      {loaded && (
+        <Box
+          dangerouslySetInnerHTML={{ __html: html }}
+          sx={{
+            marginBottom: spacing(5),
+            fontFamily: 'Titillium Web',
+            fontWeight: 400,
+            fontSize: '18px',
+            color: palette.text.primary,
+          }}
+        ></Box>
+      )}
+    </>
```

Comment:
```suggestion
    loaded && (
      <Box
        dangerouslySetInnerHTML={{ __html: html }}
        sx={{
          marginBottom: spacing(5),
          fontFamily: 'Titillium Web',
          fontWeight: 400,
          fontSize: '18px',
          color: palette.text.primary,
        }}
      />
    )
```
if you want to avoid the fragment, you can do this.